[2023-04-12T04:21:16.345+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T04:21:16.407+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T04:21:16.409+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T04:21:16.411+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2023-04-12T04:21:16.412+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T04:21:16.444+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): task_leaders> on 2023-04-11 00:00:00+00:00
[2023-04-12T04:21:16.560+0000] {standard_task_runner.py:55} INFO - Started process 98 to run task
[2023-04-12T04:21:16.786+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_extraction_dag', 'task_leaders', 'scheduled__2023-04-11T00:00:00+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp_w_ilxpj']
[2023-04-12T04:21:16.951+0000] {standard_task_runner.py:83} INFO - Job 259: Subtask task_leaders
[2023-04-12T04:21:17.692+0000] {task_command.py:389} INFO - Running <TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [running]> on host 392d142fd52f
[2023-04-12T04:21:18.330+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_extraction_dag
AIRFLOW_CTX_TASK_ID=task_leaders
AIRFLOW_CTX_EXECUTION_DATE=2023-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-11T00:00:00+00:00
[2023-04-12T04:21:18.350+0000] {logging_mixin.py:137} INFO - Fetching for leaders started
[2023-04-12T04:21:19.309+0000] {logging_mixin.py:137} INFO - Fetching for leaders ended
[2023-04-12T04:21:19.311+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-12T04:21:19.331+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_extraction_dag, task_id=task_leaders, execution_date=20230411T000000, start_date=20230412T042116, end_date=20230412T042119
[2023-04-12T04:21:19.386+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-12T04:21:19.425+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-04-12T05:46:44.624+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T05:46:44.643+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T05:46:44.644+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T05:46:44.646+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2023-04-12T05:46:44.647+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T05:46:44.667+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): task_leaders> on 2023-04-11 00:00:00+00:00
[2023-04-12T05:46:44.720+0000] {standard_task_runner.py:55} INFO - Started process 207 to run task
[2023-04-12T05:46:44.769+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_extraction_dag', 'task_leaders', 'scheduled__2023-04-11T00:00:00+00:00', '--job-id', '395', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp9o51ly2g']
[2023-04-12T05:46:44.785+0000] {standard_task_runner.py:83} INFO - Job 395: Subtask task_leaders
[2023-04-12T05:46:45.412+0000] {task_command.py:389} INFO - Running <TaskInstance: data_extraction_dag.task_leaders scheduled__2023-04-11T00:00:00+00:00 [running]> on host 392d142fd52f
[2023-04-12T05:46:45.868+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_extraction_dag
AIRFLOW_CTX_TASK_ID=task_leaders
AIRFLOW_CTX_EXECUTION_DATE=2023-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-11T00:00:00+00:00
[2023-04-12T05:46:45.899+0000] {logging_mixin.py:137} INFO - Fetching for leaders started
[2023-04-12T05:46:48.444+0000] {logging_mixin.py:137} INFO - Fetching for leaders ended
[2023-04-12T05:46:48.446+0000] {python.py:177} INFO - Done. Returned value was: <bound method DataFrame.to_dict of     displayValue   value  ...               name shortDisplayName
0           4823  4823.0  ...       passingYards             PYDS
1           4740  4740.0  ...       passingYards             PYDS
2           4633  4633.0  ...       passingYards             PYDS
3           4581  4581.0  ...       passingYards             PYDS
4           4544  4544.0  ...       passingYards             PYDS
..           ...     ...  ...                ...              ...
245            7     7.0  ...  rushingTouchdowns               TD
246            7     7.0  ...  rushingTouchdowns               TD
247            7     7.0  ...  rushingTouchdowns               TD
248            7     7.0  ...  rushingTouchdowns               TD
249            6     6.0  ...  rushingTouchdowns               TD

[250 rows x 8 columns]>
[2023-04-12T05:46:48.722+0000] {xcom.py:629} ERROR - Object of type method is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-04-12T05:46:48.731+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 2298, in xcom_push
    XCom.set(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/xcom.py", line 234, in set
    value = cls.serialize_value(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type method is not JSON serializable
[2023-04-12T05:46:48.800+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=data_extraction_dag, task_id=task_leaders, execution_date=20230411T000000, start_date=20230412T054644, end_date=20230412T054648
[2023-04-12T05:46:48.902+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 395 for task task_leaders (Object of type method is not JSON serializable; 207)
[2023-04-12T05:46:48.956+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-12T05:46:49.054+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
